.github/workflows/ci-aarchxx.yml

```diff
@@ -209,6 +209,7 @@ jobs:
         cd /tmp
```

Comment:
> Fixes the problem of NDK setup failing due to ANDROID_NDK_ROOT point a wrong position. The GitHub VM system 

grammar: s/point a wrong position/pointing to the wrong path/

> preset ANDROID_NDK_ROOT to a location which not include the necessary resources that the setup process needs. 

grammar: s/not/does not/

Reply:
Done.